core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -59,6 +61,10 @@ private[spark] abstract class WebUI(
   def getTabs: Seq[WebUITab] = tabs
   def getHandlers: Seq[ServletContextHandler] = handlers
 
+  def getDelegatingHandlers: Seq[DelegatingServletContextHandler] = {
+    handlers.map(handler => new DelegatingServletContextHandler(handler))
```

Comment:
`.map(new DelegatingServletContextHandler(_))` (or use the `{ ... }` syntax)

Reply:
Updated